dask_jobqueue/tests/test_oar.py

```diff
@@ -28,28 +28,55 @@ def test_header():
         assert "#OAR -q regular" in cluster.job_header
         assert "#OAR -t besteffort" in cluster.job_header
 
-    with OARCluster(cores=4, memory="8GB") as cluster:
```

Comment:
I'm not sure if you left some tests checking the generated job_script without using `oar_mem_core_property_name` kwarg?